src/frontend/sharedComponents/Button.tsx

```diff
@@ -1,51 +1,39 @@
 /* eslint-disable react-native/no-unused-styles */
-// @flow
 import * as React from "react";
-import PropTypes from "prop-types";
-import { StyleSheet, View } from "react-native";
-import Text from "./Text";
-import { TouchableNativeFeedback } from "../sharedComponents/Touchables";
+import { GestureResponderEvent, StyleSheet, View } from "react-native";
 
 import { VERY_LIGHT_BLUE } from "../lib/styles";
 import type { ViewStyleProp } from "../types";
 
-// Fix warning pending https://github.com/kmagiera/react-native-gesture-handler/pull/561/files
-TouchableNativeFeedback.propTypes = {
-  ...TouchableNativeFeedback.propTypes,
-  background: PropTypes.object,
-};
+import Text from "./Text";
+import { TouchableNativeFeedback } from "./Touchables";
 
-type Props = {
-  onPress: (SyntheticEvent<>) => any,
-  children: string | React.Node,
-  variant?: "contained" | "outlined" | "text",
-  color?: "dark" | "light",
-  style?: ViewStyleProp,
-  fullWidth?: boolean,
-  testID?: string,
-  disabled?: boolean,
-  size?: "small" | "medium",
-};
+type ColorScheme = "dark" | "light";
+type Variant = "contained" | "outlined" | "text";
+
+interface Props {
+  children: React.ReactNode;
+  color?: ColorScheme;
+  disabled?: boolean;
+  fullWidth?: boolean;
+  onPress: (event: GestureResponderEvent) => void;
+  style?: ViewStyleProp;
+  testID?: string;
+  variant?: Variant;
+}
 
 const Button = ({
```

Comment:
Some changes to this component were merged into `develop` recently so make sure to update this to reflect that. Think you could also copy one of the implementations from my PRs ([this](https://github.com/digidem/mapeo-mobile/blob/b9bfe59b8b5ff74270e440ae3ea8e04d4183e054/src/frontend/sharedComponents/Button.tsx) or [this](https://github.com/digidem/mapeo-mobile/blob/7c64d6105f86c00c499e9708394c9b866997b637/src/frontend/sharedComponents/Button.tsx))